configure.ac

```diff
@@ -46,6 +46,20 @@ AC_CHECK_LIB([pmi], [PMI_Allgather],
 	     AM_CONDITIONAL([HAVE_CRAY_PMI], [true]),
 	     AM_CONDITIONAL([HAVE_CRAY_PMI], [false]))
 
+AC_ARG_WITH([profiler],
+            AC_HELP_STRING([--with-profiler], [Specify libprofiler location - default NO]),
+            [AS_IF([test -d $withval/lib64], [profiler_libdir="lib64"], [profiler_libdir="lib"])
+             CPPFLAGS="-I $withval/include $CPPFLAGS"
+             CPPFLAGS="-I $withval $CPPFLAGS"
```

Comment:
I'll look into it. I recall something funky about the gperftools paths but it might just a be a copy paste error.